src/puppetlabs/trapperkeeper/bootstrap.clj

```diff
@@ -58,29 +71,18 @@
       (string/replace #"(?:#|;).*$" "")
       (string/trim)))
 
-(schema/defn find-bootstraps-from-path :- BootstrapFiles
+(schema/defn find-bootstraps-from-path :- [schema/Str]
   "Given a path, return a list of .cfg files found there.
    - If the path leads directly to a file, return a list with a single item.
    - If the path leads to a directory, return a list of any .cfg files found there.
    - If the path doesn't lead to a file or directory, attempt to load a file from
    a URI (for files in jars)"
   [config-path :- schema/Str]
-  (cond
-    (fs/file? config-path) [(fs/file config-path)]
-    (fs/directory? config-path) (fs/glob (fs/file config-path "*.cfg"))
-    :else (try
-            [(io/input-stream (URI. config-path))]
-            (catch Exception ignored
-              ;; If we can't find a file or a directory, and loading the URI
-              ;; fails, we give up and throw an exception.
-              ;; Don't wrap and re-throw here, as `ignored` may be misleading
-              ;; (in the case of a normal path, it is useless - there is no
-              ;; reason to mess with URIs)
-              (throw (IllegalArgumentException.
-                      (str "Specified bootstrap config file does not exist: '"
-                           config-path "'")))))))
-
-(schema/defn ^:private config-from-cli :- (schema/maybe BootstrapFiles)
+  (if (fs/directory? config-path)
+    (map str (fs/glob (fs/file config-path "*.cfg")))
```

Comment:
Does this implementation still handle the ability to load a file from a URI / file inside of a jar?